recipes/pbsim/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: pbsim
+  version: "1.0.3"
+
+about:
+  home: https://code.google.com/archive/p/pbsim/
+  license: GPLv2
+  summary: "PBSIM simulates PacBio reads"
+
+source:
+  url: https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/pbsim/pbsim-1.0.3.tar.gz
+
+requirements:
+  build:
+    - gcc
```

Comment:
Did you check that no linking against libgcc happens? You can do so with conda inspect.